internal/clients/http/image_builder/image_client.go

```diff
@@ -44,66 +45,63 @@ func newImageBuilderClient(ctx context.Context) (clients.ImageBuilder, error) {
 }
 
 func (c *ImageBuilderClient) Ready(ctx context.Context) error {
+	logger := ctxval.Logger(ctx)
+
 	resp, err := c.client.GetReadiness(ctx, headers.AddImageBuilderIdentityHeader)
 	if err != nil {
-		ctxval.Logger(ctx).Error().Err(err).Msgf("Readiness request failed for image builder: %s", err.Error())
+		logger.Error().Err(err).Msgf("Readiness request failed for image builder: %s", err.Error())
 		return err
 	}
 	defer resp.Body.Close()
-	if !parsing.IsHTTPStatus2xx(resp.StatusCode) {
-		ctxval.Logger(ctx).Warn().Msgf("Readiness response from image builder: %d", resp.StatusCode)
-		return ClientErr
+
+	err = http.HandleHTTPResponses(ctx, resp.StatusCode)
+	if err != nil {
+		return fmt.Errorf("ready call: %w", err)
 	}
 	return nil
 }
 
 func (c *ImageBuilderClient) GetAWSAmi(ctx context.Context, composeID string) (string, error) {
-	ctxval.Logger(ctx).Info().Msgf("Getting AMI of image %v", composeID)
+	logger := ctxval.Logger(ctx)
+	logger.Trace().Msgf("Getting AMI of image %v", composeID)
+
 	imageStatus, err := c.fetchImageStatus(ctx, composeID)
 	if err != nil {
 		return "", err
 	}
-	ctxval.Logger(ctx).Debug().Msgf("Verifying AWS type")
+
+	logger.Trace().Msgf("Verifying AWS type")
 	if imageStatus.Type != UploadTypesAws {
-		ctxval.Logger(ctx).Warn().Err(err).Msg("Image is not AWS type")
-		return "", BadImageTypeErr
+		return "", fmt.Errorf("%w: expected image type AWS", UnknownImageTypeErr)
 	}
-	awsStatus, ok := imageStatus.Options.(map[string]interface{})
+	awsStatus, ok := imageStatus.Options.(map[string]interface{})["ami"]
```

Comment:
Lets rename the var, if we want to change content of it? :)

Reply:
Renamed.